.travis/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
I'm not sure why I shouldn't add it into `after_script.sh`. If `after_script.sh` return a non-0 exit code, Travis won't say the build failed. That's exactly the purpose of the `after_script` in the `.yml` file.